### PR TITLE
Added VerifyPromotionsTest

### PR DESCRIPTION
**Pull request by vitalik1985o**

### CODE REVIEW

File: src/test/java/pages/NovusPage.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        openCity.shouldBe(visible).click();
+    }
+
+    public void choseAnotherCity() {
+        anotherCity.shouldBe(visible).click();
+    }
+
+    public void visibilityPromotionsOfAnotherCity() {
+        promotionsOfAnotherCity.shouldHave(size(4));
+    }
```
Comment (Igor0187) [inline src/test/java/pages/NovusPage.java:39]: check thst elements exsist not by amount

File: src/test/java/pages/NovusPage.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    private final ElementsCollection promotionsOfAnotherCity = $$x("//*[@data-testid='offers_list']/*[@data-marker='Offer']");
+
+    public void clickNovus() {novus.shouldBe(visible).click();}
+
+    public void clickPromotionsStore() {
+        offers.shouldBe(interactable).click();
+    }
+
+    public void visibilityPromotions() {promotions.shouldHave(size(5)); }
+
```
Comment (Igor0187) [inline src/test/java/pages/NovusPage.java:28]: same, check thet element exist

**Review by Igor0187 — COMMENTED** [review]
update test)

File: src/test/java/pages/SignInPage.java
Comment (Igor0187) [inline src/test/java/pages/SignInPage.java:1]: Rename Class (HomePage)

File: src/test/java/tests/SignInPageTest.java
Comment (Igor0187) [inline src/test/java/tests/SignInPageTest.java:1]: Rename test (SingInTest)

File: src/test/java/pages/SignInPage.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import common.TestInit;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class SignInPage extends TestInit {
+
+    private static final SelenideElement CLICK_LOGIN_BUTTON = $x("//span[text()='Увійти']");
+    private static final SelenideElement ENTER_PHONE = $x("//input[@id ='phone']");
+    private static final SelenideElement ENTER_PASSWORD = $x("//input[@id ='password']");
```
Comment (Igor0187) [inline src/test/java/pages/SignInPage.java:13]:
```
    private static final SelenideElement PHONE_FIELD = $x("//input[@id ='phone']");
    private static final SelenideElement PASSWORD_FIELD= $x("//input[@id ='password']");
```

File: src/test/java/pages/SignInPage.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    private static final SelenideElement CLICK_SUBMIT_BUTTON = $x("//button[@data-marker='Submit']");
+    private static final SelenideElement IS_LOG_BUTTON_DISPLAYED = $x("//div/span[text()='Акаунт']");
```
Comment (Igor0187) [inline src/test/java/pages/SignInPage.java:15]: ``
    private static final SelenideElement SUBMIT_BUTTON = $x("//button[@data-marker='Submit']");
private static final SelenideElement ACCOUNT_ICON = $x("//div/span[text()='Акаунт']");
``

File: src/test/java/tests/SignInPageTest.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+import static utils.ConfigProvider.PHONE;
+
+public class SignInPageTest extends TestInit {
+    SignInPage signInPage = new SignInPage();
+
+    @Test(description = "Positive Test-Verify Login Functionality")
+    @Description("SF2-26")
+    public void loginPage() {
+
+        open(BASE_URL);
```
Comment (Igor0187) [inline src/test/java/tests/SignInPageTest.java:19]: Move open(BASE_URL); to TestInit

**Review by Igor0187 — APPROVED** [review]

File: src/test/java/pages/NovusPage.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+import common.BasePage;
+
+import static com.codeborne.selenide.CollectionCondition.sizeGreaterThan;
+import static com.codeborne.selenide.Condition.interactable;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.*;
+
+public class NovusPage extends BasePage {
+
+    private static final SelenideElement NOVUS_MARKET = $x("//a[@href='https://novus.zakaz.ua/uk/']");
```
Comment (Igor0187) [inline src/test/java/pages/NovusPage.java:14]: Move NOVUS_MARKET  to HomePage

File: src/test/java/pages/NovusPage.java [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    private static final SelenideElement OFFERS_OF_MARKET = $("[data-marker='offers widget']");
+    private static final SelenideElement OPEN_CITY = $x("//*[@class='SelectStyled css-2b097c-container']");
+    private static final ElementsCollection PROMOTIONS_OF_CITY = $$x("//*[@data-testid='offers_list']/*[@data-marker='Offer']");
+    private static final ElementsCollection PROMOTIONS_OF_SPECIFIC_CITY = $$x("//*[@data-testid='offers_list']/*[@data-marker='Offer']");
+    private static final String SPECIFIC_CITY = "//*[contains(text(), '%s')]";
+
+
+    public void clickNovus() {
+        NOVUS_MARKET.shouldBe(visible).click();
+    }
```
Comment (Igor0187) [inline src/test/java/pages/NovusPage.java:24]: Move method to HomePage

File: src/test/java/pages/NovusPage.java [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+import static com.codeborne.selenide.CollectionCondition.sizeGreaterThan;
+import static com.codeborne.selenide.Condition.interactable;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.*;
+
+public class NovusPage extends BasePage {
+
+    private static final SelenideElement NOVUS_MARKET = $x("//a[@href='https://novus.zakaz.ua/uk/']");
+    private static final SelenideElement OFFERS_OF_MARKET = $("[data-marker='offers widget']");
```
Comment (Igor0187) [inline src/test/java/pages/NovusPage.java:15]: Change locator to xpath 
```suggestion
    private static final SelenideElement MARKET_OFFERS = $x("//a[@data-marker='offers widget']");
```

File: src/test/java/pages/NovusPage.java [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+    private static final SelenideElement OPEN_CITY = $x("//*[@class='SelectStyled css-2b097c-container']");
```
Comment (Igor0187) [inline src/test/java/pages/NovusPage.java:16]:
```suggestion
    private static final SelenideElement OPEN_CITY = $x("//div[contains(@class,'SelectStyled ')]");
```

File: src/test/java/pages/NovusPage.java [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+        NOVUS_MARKET.shouldBe(visible).click();
+    }
+
+    public void clickPromotionsStore() {
+        OFFERS_OF_MARKET.shouldBe(interactable).click();
+    }
+
+    public void visibilityPromotions() {
+        PROMOTIONS_OF_CITY.shouldHave(sizeGreaterThan(0));
+    }
```
Comment (Igor0187) [inline src/test/java/pages/NovusPage.java:32]: This method doesn't return anything
```suggestion
   public boolean isPromotionsExists() {
        return PROMOTIONS_OF_CITY.shouldHave(Condition.sizeGreaterThan(0)).size() > 0;
    }
```

File: src/test/java/pages/NovusPage.java [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+        MARKET_OFFERS.shouldBe(interactable).click();
+    }
+
+    public void clickOpenCity() {
+        OPEN_CITY.shouldBe(visible).click();
+    }
+
+    public void visibilityPromotionsOfAnotherCity() {
+        PROMOTIONS_OF_SPECIFIC_CITY.shouldHave(sizeGreaterThan(0));
+    }
```
Comment (Igor0187) [inline src/test/java/pages/NovusPage.java:33]:
```suggestion
    public boolean visibilityPromotionsOfAnotherCity() {
       return PROMOTIONS_OF_SPECIFIC_CITY.shouldHave(sizeGreaterThan(0));
    }
```

**Review by Igor0187 — APPROVED** [review]

**Review by NK20018 — APPROVED** [review]

**Pull request merged**